### ISSUE TITLE
It’s time to take over, no longer can we stand on the sidelines and let other coins pass us!

### ISSUE THREAD

**Issue by illustriousdoge** [dogecoin/dogecoin#2054]
# Feature Request

Does anyone think Dogecoin needs to combine with some things, in order to be successful? we can be the main coin like bitcoin,  we can have a meme coin under doge, ass finance or one of the other ones we know coins are aiming to be the doge killer so if Doge has a secondary coin we can focus on the big ones like BTC ETH BNB etc. we are not trying to be put in the madness coin creation category. Doge also can have a gaming platform where you can use/earn real doge maybe even bet doge like, WSGtoken, the WSB twitter account with 1million followers is already pushing for them to use doge it can be completed. Lastly have a NFT trading creating platform it can be popular memes too, maybe tcgcoin which is anime based but I’m sure they could transfer to other things if they are willing if not then Doge can create our own thing for any type of NFT trading platform if we combine all 3 of those together with dogearmy it is over for the other coins. I am afraid this way currently we cannot successfully take off due to all the other coins even with elon and others helping we gotta take people by storm and show who the REAL DOGE army is!!! 

**Describe the Feature Request**

**Describe Preferred Solution**

**Describe Alternatives**
<!-- A clear and concise description of any alternative solutions or features you've considered. --

**Comment by simonmadsen** [comment]
the doge should be proof of stake or similar first? preferred

**Comment by illustriousdoge** [comment]
> the doge should be proof of stake or similar first? preferred

I don’t know exactly would similar first preferred is, but I think in this instance maybe proof of work would be better the problem is Elon initially dropped bitcoin because of the energy efficiency so I don’t think the community would go with Proof of work. according to the two frowns on my post people don’t like being aggressive and forefront because this is community-based and they are about to do good every day, well while we are doing good every day other coins are attacking us like safe moon and shiba inu if we sit back and allow them to continue what they doing they’ve already taken some more fan base I want to put a stop to it or at least take initial steps to go forward. Doge is already less energy efficient than bitcoin if we can increase some of the energy we use I believe including other coins could be possible so your idea of proof of stake more than likely will be the route to go.

**Comment by m-luck** [comment]
I support this. Can you propose technical ways for us to achieve this? Not claiming that you need to already know anything to participate. Maybe a week or two of reading and we can discuss technically detailed solutions here and make them happen.

**Comment by illustriousdoge** [comment]
> I support this. Can you propose technical ways for us to achieve this? Not claiming that you need to already know anything to participate. Maybe a week or two of reading and we can discuss technically detailed solutions here and make them happen.

https://twitter.com/nftblockchain/status/1396998408829816834?s=21
is the only technical answer that I am able to give an already established NFTtrading platform that would be willing to integrate and/or convert to a Doge based NFT platform. A lot of major supporters with followers already follow this NFT company they could possibly in the works already. honestly I would like to see a little less meme content and a little more on what we’re doing to improve and get new fans. aside from Elon musk of course Who is working at the top to push this.

as far as the other two things no I do not unfortunately I’ve already sent this same statement to the creator of Dogecoin where you can contact him directly if you use his website, I asked your question as well is any of the ideas actually a possibility to integrate with Doge. Do they need to be standalone projects and we have to include some third-party to make sure everything transitions smooth i’m not a developer I’m not very good with block chain technology I just see what is going on and what is trending and I know that we need to catch up to everything that is rolling out including tokenamics. wayback in 2013 there was just no way to know all the changes the future held but once again I’m gonna at just something that you can push a button and in.

Long story short is I am spitting out ideas without the full substance to know if all of it is possible I know definitely some of it is.

**Comment by patricklodder** [comment]
Hi! Sounds interesting! Keep us updated of your progress of implementing this, on reddit: https://www.reddit.com/r/dogecoindev/

**Comment by illustriousdoge** [comment]
![8B6DFA81-832D-4186-AFD5-61EDAC9C4A92](https://user-images.githubusercontent.com/84756453/120194344-1ccf0b80-c1e3-11eb-8041-da9e6f62105d.png)

exactly like I said we need to combine different things to make dogecoin better

**Comment by simonmadsen** [comment]
> ![8B6DFA81-832D-4186-AFD5-61EDAC9C4A92](https://user-images.githubusercontent.com/84756453/120194344-1ccf0b80-c1e3-11eb-8041-da9e6f62105d.png)
> 
> exactly like I said we need to combine different things to make dogecoin better

original tweet please

**Comment by illustriousdoge** [comment]
https://twitter.com/wsbchairman/status/1399325049472049154?s=21

there is the original tweet 

![FC445BBE-D8CD-4007-B872-5107252940B6](https://user-images.githubusercontent.com/84756453/120195145-083f4300-c1e4-11eb-8f06-4a0434c33791.png)
